kinto/plugins/hawk/__init__.py

```diff
@@ -0,0 +1,8 @@
+
+
```

Comment:
you can import `HawkAuth` here, so that we can have `from kinto.plugins.hawk import HawkAuth`